firefox-ios/firefox-ios-tests/Tests/XCUITests/BookmarksTests.swift

```diff
@@ -11,6 +11,11 @@ let urlLabelExample_3 = "Example Domain"
 let url_3 = "localhost:\(serverPort)/test-fixture/test-example.html"
 
 class BookmarksTests: BaseTestCase {
+    override func tearDown() {
+        XCUIDevice.shared.orientation = .portrait
```

Comment:
why do we need this now? tests should all start in portrait..

Reply:
because i have added validation in landscape mode in my test and in case this fails in this mode, the teardown will bring the mode back to portrait